dockerfiles/run/guice/cassandra-rabbitmq-ldap/destination/conf/blob.properties

```diff
@@ -1,17 +1,118 @@
+# ============================================= BlobStore Implementation ==================================
+# Choose your BlobStore implementation
+# Compulsory, allowed values are: cassandra, objectstorage, union
+# union is using both objectstorage as the current BlobStore & cassandra as the legacy BlobStore
 implementation=objectstorage
 
-# Configuration for swift BlobStore
+# ============================================== ObjectStorage ============================================
+
+# ========================================= ObjectStorage Encryption ======================================
+# Codec for blob contents
+# Compulsory, allowed values are: DEFAULT, AES256
+# DEFAULT: no encryption
```

Comment:
so DEFAULT encryption is no encryption?
Please rename the section to ObjectStorage Codec